beacon-chain/node/node.go

```diff
@@ -662,3 +669,109 @@ func (b *BeaconNode) registerArchiverService() error {
 	})
 	return b.services.RegisterService(svc)
 }
+
+func loadChainConfigFile(chainConfigFileName string) {
```

Comment:
Can `loadChainConfigFile` and `replaceHexStringWithYAMLFormat` not inside `node.go`?

These are only helper methods just for loading custom chain configs.

Reply:
moved